lib/pure/unittest.nim

```diff
@@ -550,7 +550,7 @@ template test*(name, body) {.dirty.} =
         defer: testTeardownIMPL()
       body
 
-    except:
+    except CatchableError:
```

Comment:
```suggestion
    except CatchableError, Defect:
```
Unit test are one of the rare places where catching defect makes sense

Reply:
Actually they are not completely equal, bare except can catch backend exceptions.

Reply:
What to do then... do we just silence the warning?

Reply:
> catching defect makes sense

might even make sense to `catch Exception` in such cases - ie the undefined behavior that follows doesn't really matter

> catch backend exceptions

only in c++ mode though, and not in v2 with goto exceptions.. ?

Reply:
I updated it to catch `Exception` directly... and so be it. Let me know if we should update the code, but there are not many other alternatives.